.setup/vagrant/db_inserts.sql

```diff
@@ -4,25 +4,25 @@ INSERT INTO sections_registration(sections_registration_id)
     VALUES (1);
 
 -- Add all instructors / TAs, administrators
-INSERT INTO users (user_id, user_firstname, user_lastname, user_email, user_group, registration_section, rotating_section)
-    VALUES ('developer', 'Developer', 'Jackson', 'instructor@email.edu', 0, NULL, NULL);
+INSERT INTO users (user_id, user_password, user_firstname, user_lastname, user_email, user_group, registration_section, rotating_section)
+    VALUES ('developer', '$2y$10$DuPIEHfBo5ka582lxrY5Au0roVsH3x7DAuoIVKHKrlM6wYu7zHGWG', 'Developer', 'Jackson', 'instructor@email.edu', 0, NULL, NULL);
```

Comment:
so its just for testing, but generate these passwords on installation and distribute as needed instead of hardcoding something?